api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -63,31 +73,98 @@ public void incrementLep(){
     }
 
     public void addSavingThrowLevelOne(ClassPg classPg){
-
         String stringSavingThrow = classPg.getSavingThrow();
 
-            double bonusFortitude;
-            if(stringSavingThrow.charAt(0) == ValueEnum.HIGH.getValueEnum().charAt(0)){
-                bonusFortitude = 2.5;
-            } else{bonusFortitude = 0;}
-            this.savingThrow.setFortitude(this.savingThrow.getFortitude()+bonusFortitude);
-
-            double bonusReflex;
-            if(stringSavingThrow.charAt(1) == ValueEnum.HIGH.getValueEnum().charAt(0)){
-                bonusReflex = 2.5;
-            } else{bonusReflex = 0;}
-            this.savingThrow.setReflex(this.savingThrow.getReflex()+bonusReflex);
-
-            double bonusWill;
-            if(stringSavingThrow.charAt(2) == ValueEnum.HIGH.getValueEnum().charAt(0)){
-                bonusWill = 2.5;
-            } else{bonusWill = 0;}
-            this.savingThrow.setWill(this.savingThrow.getWill()+bonusWill);            
-        }
+        double bonusFortitude;
+        if(stringSavingThrow.charAt(0) == ValueEnum.HIGH.getValueEnum().charAt(0)){
+            bonusFortitude = 2.5;
+        } else{bonusFortitude = 0;}
+        this.savingThrow.setFortitude(this.savingThrow.getFortitude()+bonusFortitude);
+
+        double bonusReflex;
+        if(stringSavingThrow.charAt(1) == ValueEnum.HIGH.getValueEnum().charAt(0)){
+            bonusReflex = 2.5;
+        } else{bonusReflex = 0;}
+        this.savingThrow.setReflex(this.savingThrow.getReflex()+bonusReflex);
+
+        double bonusWill;
+        if(stringSavingThrow.charAt(2) == ValueEnum.HIGH.getValueEnum().charAt(0)){
+            bonusWill = 2.5;
+        } else{bonusWill = 0;}
+        this.savingThrow.setWill(this.savingThrow.getWill()+bonusWill);            
+    }
 
     public void incementSavingThrow() {
         this.savingThrow.setFortitude(this.savingThrow.getFortitude()+0.5);
         this.savingThrow.setReflex(this.savingThrow.getReflex()+0.5);
         this.savingThrow.setWill(this.savingThrow.getWill()+0.5);
     }
+
+    public void setSkillsTruePgArray(Set<Skills> availableSkills) {
+        for(Skills skill : availableSkills){
+            for(ClassSkills classSkill : classSkills){
+                if(skill.getId() == classSkill.getIdSkill()){
+                    classSkill.setClassSkill(true);
+                }
+            }
+        }
+    }
+
+    public void createSkillsArray(List<Skills> skillsList) {
+        boolean check = true;
+        if(classSkills.isEmpty()){
+            check = false;
+        }
+        if(check == false){
```

Comment:
nigdy nie porównujemy do true false. To głupio wyglada. mozesz tu dac if(!check)